System/DataModule.h

```diff
@@ -110,6 +121,12 @@ namespace RTE {
 		/// <returns>Whether this DataModule is considered a faction or not.</returns>
 		bool IsFaction() const { return m_IsFaction; }
 
+		/// <summary>
+		/// Gets whether this DataModule is considered a market.
+		/// </summary>
+		/// <returns>Whether this DataModule is considered a market or not.</returns>
+		bool IsMarket() const { return m_IsMarket; }
```

Comment:
I dunno if `market` is the best term here. `Merchant` might be better? `Independent`? `IndependentMarket`?

Reply:
`IsAmazonPrime`.
Merchant works I guess.

Reply:
lel
Yeah, let's go with that I guess